engines/wintermute/base/base_keyboard_state.cpp

```diff
@@ -334,11 +345,225 @@ enum VKeyCodes {
 	kVkNumLock    = 144,
 	kVkScroll     = 145
 
-    //TODO: shift, ctrl, menu, etc...
+	//TODO: shift, ctrl, menu, etc...
+};
+
+#ifdef ENABLE_FOXTAIL
+enum FoxtailVKeyCodes {
+	kFtVkBack       = 8,
+	kFtVkTab        = 9,
+	kFtVkClear      = 1073741980,
+	kFtVkReturn     = 13,
+	kFtVkPause      = 1073741896,
+	kFtVkCapital    = 1073741881,
+	kFtVkEscape     = 27,
+	kFtVkSpace      = 32, //printable
+
+	kFtVkPrior      = 1073741899,
+	kFtVkNext       = 1073741902,
+	kFtVkEnd        = 1073741901,
+	kFtVkHome       = 1073741898,
+	kFtVkLeft       = 1073741904,
+	kFtVkUp         = 1073741906,
+	kFtVkRight      = 1073741903,
+	kFtVkDown       = 1073741905,
+	kFtVkPrint      = 1073741894,
+	kFtVkInsert     = 1073741897,
+	kFtVkDelete     = 127,
+
+	kFtVkF1         = 1073741882,
+	kFtVkF2         = 1073741883,
+	kFtVkF3         = 1073741884,
+	kFtVkF4         = 1073741885,
+	kFtVkF5         = 1073741886,
+	kFtVkF6         = 1073741887,
+	kFtVkF7         = 1073741888,
+	kFtVkF8         = 1073741889,
+	kFtVkF9         = 1073741890,
+	kFtVkF10        = 1073741891,
+	kFtVkF11        = 1073741892,
+	kFtVkF12        = 1073741893,
+
+	kFtVkNumLock    = 1073741907,
+	kFtVkScroll     = 1073741895
```

Comment:
These all look to be ORed with 1073741824 (01000000000000000000000000000000). Perhaps we can just remove this bit and handle them like the other keycodes?

Reply:
Well, removing 01000000000000000000000000000000 won't help much since those constants are different from `VKeyCodes` anyway.
Compare this (subsequent codes for some keys):
```
	kVkNumLock    = 144,
	kVkScroll     = 145
```
And this (not-subsequent codes for same keys):
```
	kFtVkNumLock    = 1073741907,
	kFtVkScroll     = 1073741895
```
This means that we would need to have 2 enums anyway, even if 01000000000000000000000000000000 part is removed.